### DISCUSSION TITLE
[web] Allow direct upload to albums where user is collaborator

### DISCUSSION THREAD

**Discussion by Tuscann** [ente-io/ente#3011] [Enhancements]
### Description

1. Open https://web.ente.io/gallery
2. Login with user bai ivan
3. Open album Camera Roll where user bai ivan has right for collaborator (Camera Roll is album created from user mancho)
4. Click Upload -> File -> Choose ivan.png
5. Check albums ?? Missing album Camera Roll where to add new file

https://github.com/user-attachments/assets/47d85214-e50e-4c09-99e0-718bd2b9388a

### Version

WEB

### What product are you using?

Ente Photos

### What platform are you using?

Web - Chrome

**Comment by ua741** [comment]
(internal note) We will need to upload the album to uncategorized collection owned by the user, and then add a link to the collaborative albums. This is required because collaborator can only add symlinks to files that they own, and direct upload to album owned by others is not supported.

**Reply by Conobi** [reply]
Why the issue has been closed? I still have this issue right now!

**Reply by dev-goalz** [reply]
Created a PR for this https://github.com/ente-io/ente/pull/10278
